### PR TITLE
feat(pipeline/execution): Adds get cmd for pipeline executions.

### PR DESCRIPTION
**Pull request by jtk54**

### CODE REVIEW

**Comment by jtk54** [comment]
Partly closes https://github.com/spinnaker/spinnaker/issues/3999.

File: cmd/pipeline/execution/execution.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package execution
+
+import (
+"github.com/spf13/cobra"
+"io"
```
Comment (ttomsu) [inline cmd/pipeline/execution/execution.go]: formatting

Comment (jtk54) [reply cmd/pipeline/execution/execution.go]: Yeah there were a lot of format issues, reformatted all the files.

File: util/args_reader.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		}
+
+		scanner := bufio.NewScanner(os.Stdin)
+		input := ""
+		for scanner.Scan() {
+			input = scanner.Text()
+		}
+		return input, nil
+	}
+	return args[0], nil
```
Comment (ttomsu) [inline util/args_reader.go]: Prefer to return early

File: util/cli_ui.go [hunk 3]
```diff
@@ -103,8 +103,19 @@ func (u *ColorizeUi) JsonOutput(input interface{}, outputFormat *output.OutputFo
...
 			u.Error(fmt.Sprintf("%v", err))
 		}
-		u.Output(u.colorize(string(prettyStr), u.OutputColor))
+		// unquote since go quotes the string if the bytes is a string.
+		u.Output(u.colorize(u.unquote(string(prettyStr)), u.OutputColor))
+	}
+}
+
+func (u *ColorizeUi) unquote(input string) string {
+	if len(input) > 0 && input[0] == '"' {
```
Comment (ttomsu) [inline util/cli_ui.go]: See strings.TrimPrefix and strings.TrimSuffix

File: util/args_reader.go (hunk 2 above)
Comment (jtk54) [reply util/args_reader.go]: What does that mean in context?

Comment (ttomsu) [reply util/args_reader.go]: I mean try reducing nesting/indentation by reversing the conditional to look something like:

```
if len(args) != 0 {
  return args[0], nil
}

stdin := os.Stdin
fi, err :- stdin.Stat()
...

Comment (jtk54) [reply util/args_reader.go]: Got it. Done.

**Pull request merged**